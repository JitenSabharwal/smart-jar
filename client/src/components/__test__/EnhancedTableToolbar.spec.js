/* eslint-disable no-undef */
import React from 'react'
import Enzyme, {shallow} from 'enzyme'
import EnhancedTableToolbar from '../EnhancedTableToolbar.jsx'
import Adapter from 'enzyme-adapter-react-16'

const handleDelete = (event) => {
  console.log(event)
}
const numSelected = 0
const selected = ''
Enzyme.configure({adapter: new Adapter()})
describe('EnhancedTableToolbar', () => {
  test('render', () => {
    const wrapper = shallow(<EnhancedTableToolbar
      handleDelete={handleDelete}
      numSelected={numSelected}
      selected={selected}
      tableName={'Users'}
    />)
    expect(wrapper.exists()).toBe(true)
    expect(wrapper).toMatchSnapshot()
  })
})
