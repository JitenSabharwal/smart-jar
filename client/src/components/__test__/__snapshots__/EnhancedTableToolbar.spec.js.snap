// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnhancedTableToolbar render 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(EnhancedTableToolbar)
    handleDelete={[Function]}
    numSelected={0}
    selected=""
    tableName="Users"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "classes": Object {
        "actions": "EnhancedTableToolbar-actions-4",
        "highlight": "EnhancedTableToolbar-highlight-2",
        "root": "EnhancedTableToolbar-root-1",
        "spacer": "EnhancedTableToolbar-spacer-3",
        "title": "EnhancedTableToolbar-title-5",
      },
      "handleDelete": [Function],
      "numSelected": 0,
      "selected": "",
      "tableName": "Users",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "classes": Object {
          "actions": "EnhancedTableToolbar-actions-4",
          "highlight": "EnhancedTableToolbar-highlight-2",
          "root": "EnhancedTableToolbar-root-1",
          "spacer": "EnhancedTableToolbar-spacer-3",
          "title": "EnhancedTableToolbar-title-5",
        },
        "handleDelete": [Function],
        "numSelected": 0,
        "selected": "",
        "tableName": "Users",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
